test/bigtest/interactors/order-edit-page.js

```diff
@@ -10,8 +13,34 @@ import {
   value = value();
 }
 
+@interactor class OrderTypeSelect {
+  static defaultScope = 'select[name="orderType"]';
+  value = value();
+}
+
+@interactor class CreateOrderButton {
+  static defaultScope = '[type="submit"]';
+  isButton = is('button');
+}
+
+@interactor class VendorInput {
```

Comment:
imo you can use just `new Interactor(selector)` instead of new class creation

Reply:
Updated